ts/features/ingress/store/selectors/index.ts

```diff
@@ -0,0 +1,6 @@
+import { GlobalState } from "../../../../store/reducers/types";
+
+const ingressScreenSelector = (state: GlobalState) => state.ingress;
```

Comment:
What about insert the new `ingress` state as a children of `features` like we did for `mixpanel` new state?

Reply:
It makes sense, I will do! Thanks for the suggestion!